.gitignore

```diff
@@ -15,3 +15,6 @@ coverage
 # test output
 test/**/out
 .DS_Store
+
+# IDE
+.idea
```

Comment:
You can do this in a global gitignore file. But oh well, we've seen this so many times. I want to take it in @arunoda